layout.tt

```diff
@@ -59,6 +61,7 @@
             <ul class="dropdown-menu">
               <li><a href="[%root%]">NixOS</a></li>
               <li><a href="[%root%]nix">Nix</a></li>
+              <li><a href="[%root%]nix-pills">Nix Pills</a></li>
```

Comment:
It could be, I wasn't sure which top level project it would belong to for the menu, and was worried about it being confusing to click docs in nix/nixpkgs/nixos and end up in a different section. Then I put it in its own section. What do you recommend?